app/main.py

```diff
@@ -16,11 +17,13 @@ class Config:
     _DEFAULTS = {
         'DOWNLOAD_DIR': '.',
         'AUDIO_DOWNLOAD_DIR': '%%DOWNLOAD_DIR',
+        'CUSTOM_DIRS': 'true',
+        'CREATE_DIRS': 'false',
         'STATE_DIR': '.',
         'URL_PREFIX': '',
         'OUTPUT_TEMPLATE': '%(title)s.%(ext)s',
         'OUTPUT_TEMPLATE_CHAPTER': '%(title)s - %(section_number)s %(section_title)s.%(ext)s',
-        'YTDL_OPTIONS': '{}',
+        'YTDL_OPTIONS': '{}'

```

Comment:
why the change? (the comma is there to make the diff of adding a new variable at the end a bit shorter)